localstack-core/localstack/services/s3/provider.py

```diff
@@ -708,17 +717,18 @@ def get_bucket_location(
         Botocore implements this hack for parsing the response in `botocore.handlers.py#parse_get_bucket_location`
         """
         store, s3_bucket = self._get_cross_account_bucket(context, bucket)
+        bucket_region = s3_bucket.bucket_region
```

Comment:
nit: this could be move inside the if block as it doesn't seem to be used otherwise